store/interfaces.go

```diff
@@ -30,6 +30,6 @@ type LocalStore interface {
 
 type RemoteStore interface {
 	MetadataStore
-	PublicKeyStore
+	//	PublicKeyStore
```

Comment:
Good catch! Yeah, I need to put that back in. Need to implement GetKey for the MemoryStore